Formula/kaitai-struct-compiler.rb

```diff
@@ -3,18 +3,17 @@ class KaitaiStructCompiler < Formula
   homepage "http://kaitai.io/"
   url "https://bintray.com/artifact/download/kaitai-io/universal/0.8/kaitai-struct-compiler-0.8.zip"
   sha256 "545fc10e134db2901cad8817be1b440fca6f2bad8b92b2948ebe0647f3ffa2c9"
+  revision 1
 
   bottle :unneeded
 
   depends_on :java => "1.8+"
 
   def install
     libexec.install Dir["lib/*"]
+    bin.install "bin/kaitai-struct-compiler"
 
-    (bin/"kaitai-struct-compiler").write <<~EOS
-      #!/bin/bash
-      java -cp "#{libexec}/*" io.kaitai.struct.JavaMain "$@"
-    EOS
+    inreplace bin/"kaitai-struct-compiler", /^declare -r lib_dir=.*$/, "declare -r lib_dir='#{libexec}'"
```

Comment:
Why is the inreplace necessary? If we install kaitai-struct-compiler in `libexec/bin`, and then add a symlink in `bin` is the `inreplace` still needed?

Reply:
@fridtjof ping